cmd/hologram-agent/main.go

```diff
@@ -62,10 +64,18 @@ func main() {
 		log.Debug("Using command-line remote address.")
 		config.Host = *dialAddress
 	}
+	if *accessKey != "" {
```

Comment:
so, whats weird is this is running as root, configuring it separately felt more natural than dropping a file in /root/.aws, but im not opposed to that.  i considered pushing it from the cli like the ssh key does, but didnt seem right either.